flight/Modules/Actuator/actuator.c

```diff
@@ -715,6 +827,22 @@ static void actuator_task(void* parameters)
 				MIXERSETTINGS_MIXER1VECTOR_NUMELEM,
 				1);
 
+		/* At arming time, knock all 3d actuators into 3D mode.
```

Comment:
This is a tradeoff.  Ideally the end-user sets the ESC into the correct mode.

But it is *crucial* that if we're going to send 3D commands, the actuator be 3D, because small negative commands are otherwise very high throttle.  This could be a nasty surprise in flight.

Also, if the user doesn't set the ESC into the right mode, the neutral handling is all simplified.  So, meh.